cmd/openshift-install/targets.go

```diff
@@ -18,40 +18,54 @@ import (
 	"github.com/openshift/installer/pkg/asset/metadata"
 )
 
-func newInstallConfigCmd() *cobra.Command {
-	return &cobra.Command{
+type target struct {
+	name    string
+	command *cobra.Command
+	assets  []asset.WritableAsset
+}
+
+var targets = []target{{
+	name: "Install Config",
+	command: &cobra.Command{
 		Use:   "install-config",
 		Short: "Generates the Install Config asset",
 		Long:  "",
-		RunE:  runTargetCmd(&installconfig.InstallConfig{}),
-	}
-}
-
-func newIgnitionConfigsCmd() *cobra.Command {
-	return &cobra.Command{
+	},
+	assets: []asset.WritableAsset{&installconfig.InstallConfig{}},
+}, {
+	name: "Manifests",
+	command: &cobra.Command{
+		Use:   "manifests",
+		Short: "Generates the Kubernetes manifests",
+		Long:  "",
+		RunE:  runTargetCmd(&manifests.Manifests{}, &manifests.Tectonic{}),
```

Comment:
This still duplicates the `assets` peoperty two lines below.  Not a big deal, because they're so close together, but does [building `RunE` on the fly][1] not work?

[1]: https://github.com/openshift/installer/pull/432#discussion_r223752387

Reply:
fixed, forgot to remove that.